libraries/botbuilder-core/tests/skills/test_skill_conversation_id_factory.py

```diff
@@ -0,0 +1,113 @@
+from uuid import uuid4 as uuid

```

Comment:
The copyright header is missing here. Some of the tests have it.

Reply:
We forgot to check that, thanks @ceciliaavila, I will change it shortly.

Reply:
@ceciliaavila done, thanks